examples/dataflow-data-generator/data-generator-pipeline/tests/test_get_pyarrow_translated_schema.py

```diff
@@ -0,0 +1,150 @@
+                "mode": "REQUIRED"
+            },
+            {
+                "type": "TIMESTAMP",
+                "name": "timestamp1",
+                "mode": "REQUIRED"
+            },
+            {
+                "type": "DATE",
+                "name": "date1",
+                "mode": "REQUIRED"
+            },
+            {
+                "type": "TIME",
+                "name": "time1",
+                "mode": "REQUIRED"
+            },
+            {
+                "type": "DATETIME",
+                "name": "datetime1",
+                "mode": "REQUIRED"
+            }
+        ]
+        expected_pa_schema = pa.schema(
+            [
+                pa.field(
+                    name='string1',
+                    type=pa.string(),
+                    nullable=False
+                ),
+                pa.field(
+                    name='numeric1',
+                    type=pa.int64(),
+                    nullable=True
+                ),
+                pa.field(
+                    name='integer1',
+                    type=pa.int64(),
+                    nullable=False
+                ),
+                pa.field(
+                    name='float1',
+                    type=pa.float64(),
+                    nullable=True
+                ),
+                pa.field(
+                    name='boolean1',
+                    type=pa.bool_(),
+                    nullable=False
+                ),
+                pa.field(
+                    name='timestamp1',
+                    type=pa.timestamp('ms'),
+                    nullable=False
+                ),
+                pa.field(
+                    name='date1',
+                    type=pa.date64(),
+                    nullable=False
+                ),
+                pa.field(
+                    name='time1',
+                    type=pa.time64('us'),
+                    nullable=False
+                ),
+                pa.field(
+                    name='datetime1',
+                    type=pa.timestamp('ms'),
+                    nullable=False
+                )
+            ]
+        )
+
+        pyarrow_schema = get_pyarrow_translated_schema(string_input_schema)
+        self.assertEqual(pyarrow_schema, expected_pa_schema)
+
+    def test_fix_record_for_parquet(self):
+        input_schema = [
+            {
+                "type": "TIMESTAMP",
+                "name": "timestamp1",
+                "mode": "REQUIRED"
+            },
+            {
+                "type": "DATETIME",
+                "name": "datetime1",
+                "mode": "REQUIRED"
+            },
+            {
+                "type": "DATE",
+                "name": "date1",
+                "mode": "REQUIRED"
+            }
+        ]
+
+        record = {
+            'timestamp1': '2019-03-15T20:22:28',
+            'datetime1': '2019-03-15T20:24:58',
+            'date1': '2019-03-15',
+            #  'time1': u'2012-09-08T20:20:00.00'
```

Comment:
hey @jaketf , I still don't have a test for converting the time fields to the correct timestamps because I'm a little confused on how to represent the time types. It looks like in your conversions, you append the given time with the current date to get a datetime that can be converted into an epoch timestamp? If so, how are your tests passing since the day changes but the hardcoded expected output doesn't?

Reply:
ohhhh scratch that. I see what you're doing now! You're getting the number of milliseconds since midnight for that time on any given day and just happen to use today? Is it cool if I add that same logic to the ParquetUtil?